Jenkinsfile

```diff
@@ -158,11 +158,10 @@ def yottaBuildStep(target, compilerLabel) {
               execute("gcovr -x -o junit.xml")
               execute("cppcheck --enable=all --std=c99 --inline-suppr --template=\"{file},{line},{severity},{id},{message}\" source 2> cppcheck.txt")
 
-              // Waiting for IOTTESTINF-4864
               // check if astyle is correct
-              // execute("astyle source/*.c mbed-trace/*.h --project=.astylerc")
+              execute("astyle --options=.astylerc source/*.c mbed-trace/*.h")
               // check differency
-              // execute("git diff-index --quiet HEAD")
+              execute("git diff-index --quiet HEAD")
```

Comment:
Is this output enough for the developer to see what needs to be corrected?

Reply:
yeah, maybe should not be `--quiet`..

Reply:
removed `--quiet` . I'll double check that it gives enough information if astyle is not clean..

Reply:
using now `git diff-index -p --exit-code HEAD`
results as patch format like :

```
[mbed-trace] Running shell script
+ git diff-index -p --exit-code HEAD
diff --git a/source/mbed_trace.c b/source/mbed_trace.c
index 8a19d1b..f90e0ea 100644
--- a/source/mbed_trace.c
+++ b/source/mbed_trace.c
@@ -152,7 +152,7 @@ static trace_t m_trace = {
     .mutex_lock_count = 0
 };
 
-int mbed_trace_init(void )
+int mbed_trace_init(void)
 {
     if (m_trace.line == NULL) {
         m_trace.line = MBED_TRACE_MEM_ALLOC(m_trace.line_length);
```

Better?

Reply:
much better, thanks